frame/support/test/tests/pallet.rs

```diff
@@ -282,6 +283,10 @@ pub mod pallet {
 	pub type Map2<T> =
 		StorageMap<Hasher = Twox64Concat, Key = u16, Value = u32, MaxValues = ConstU32<3>>;
 
+	#[pallet::storage]
+	pub type Map3<T> =
+		StorageMap<_, Blake2_128Concat, u32, u64, ResultQuery<Error<T>::NonExistentStorageValue>>;
```

Comment:
```suggestion
		StorageMap<_, Blake2_128Concat, u32, u64, ResultQuery<Error::<T>::NonExistentStorageValue>>;
```
This would be the standard Rust syntax, or?

Reply:
IIRC `cargo fmt` will just reformat it and remove the double colons, and to be honest, this is definitely not standard Rust syntax, as you can't specify an enum variant as a type, so what is "normal" here is up to anyone's guess.

Reply:
Uh right. But it could be a const generic :laughing:  
Anyway, just wanted to point it out in case it was an oversight.